rero_mef/views.py

```
@@ -15,15 +15,80 @@
+    targets = ("mef", "concepts_mef", "places_mef")
+
+    actions = []
+    for target in targets:
+        try:
+            if current_search_client.indices.exists_alias(name=target):
+                mapping = current_search_client.indices.get_alias(name=target)
+                for index_name in mapping.keys():
+                    actions.append({"add": {"index": index_name, "alias": alias_name}})
+                continue
+        except Exception:
+            pass
```

Ruff (0.15.6)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)